src/stack_manager.py

```diff
@@ -136,8 +138,10 @@ def DrawStack(self, scope: Scope) -> Image:
 
         for idx in reversed(deferred):
             child = images[idx]
-            child.tag.size = (images[idx+1].size[0], child.tag.size[1]) # tuple doesn't support assignment
-            images[idx] = self.drawing_manager.DrawTextMeme(child.tag, child.scoped_tags, child.children)
+            # tuple doesn't support assignment
```

Comment:
This comment feels unnecessary, but probably worth adding one for why we iterate over `deferred` in reverse order?

Reply:
I can't tell you why at this point

Reply:
I believe that means "yes, we should add one".
It's to deal with the edge case of having consecutive lookaheads, so that we construct the second one first, preventing the first one from winding up with a size that's still `("lookahead", <something>)`

Reply:
indeed. added that comment inline